### DISCUSSION TITLE
Import and Export not working for me in drift

### DISCUSSION THREAD

**Discussion by sagarbanik** [simolus3/drift#2171] [Q&A]
I was following the doc(https://drift.simonbinder.eu/docs/examples/existing_databases/).

My code:
```dart
Future<void> exportInto(AppDatabase database) async {
    final dbFolder = await getApplicationDocumentsDirectory();
    final file = File(p.join(dbFolder.path, 'db.sqlite'));

    // Make sure the directory of the target file exists
    await file.parent.create(recursive: true);

    // Override an existing backup, sqlite expects the target file to be empty
    if (file.existsSync()) {
      file.deleteSync();
    }

    await database.customStatement('VACUUM INTO ?', [file.path]);
  }
```

What i'm doing wrong? If possible please gimme a details example

**Comment by simolus3** [comment]
What exactly is not working for you? Do you get an error? Or does it just fail silently without any exception?

**Reply by sagarbanik** [reply]
Nothing happens after calling this method. I didn't see any error at all. One thing more, I didn't understand  what is custom statement

**Reply by sagarbanik** [reply]
My Goal: I want to export my local app database somewhere in the internal storage. Then I got a button in my UI named Restore Data. After clicking this button user should pick previously exported file(as db.sqlite) than import the database from picked DB file named db.sqlite.

Like we do in Mysql database import/export operations.

Any detailed example will help me. I would appreciate your reply. and Thanks for your reply again.

**Reply by sagarbanik** [reply]
The same issue was asked before. Link --> https://github.com/simolus3/drift/issues/376

But I need the solution for drift.
